yarn/src/main/scala/org/apache/spark/deploy/yarn/YarnSparkHadoopUtil.scala

```diff
@@ -415,14 +417,12 @@ object YarnSparkHadoopUtil {
    * the behavior of '%' in a .cmd file: it gets interpreted as an incomplete environment
    * variable. Windows .cmd files escape a '%' by '%%'. Thus, the correct way of writing
    * '%%p' in an escaped way is '%%%%p'.
-   *
-   * @return The correct OOM Error handler JVM option, platform dependent.
    */
-  def getOutOfMemoryErrorArgument: String = {
+  def addOutOfMemoryErrorArgument(javaOpts: ListBuffer[String]): Unit = {
```

Comment:
Hm, I think we can also mark this private[yarn]? looks like it should be given its usage. I don't believe it's intended as an API method of any kind